### ISSUE TITLE
Could this be run on Windows?

### ISSUE THREAD

**Issue by ShadowEmbrace** [seclab-ucr/INTANG#9]
Amazed by the project, I signed in just to ask if this could be run on Windows or do I have to set up a virtual platform? Thank you!

<strike>I'm no software engineer, but I'm not stupid either. If anyone would be kind enough to explain to me I'm pretty sure I'd be able to understand and<s> appreciate the help.

**Comment by gkso** [comment]
Thanks. We currently do not support Windows. You may use a Linux virtual machine running in Bridge Mode.

**Comment by OneHappyForever** [comment]
Could we run in on a server in china and have clients connect to the server and redirect connection using intang through the gfw?

**Comment by gkso** [comment]
Yes you can setup your own proxy server on a Linux machine inside China

**Comment by hejianhui** [comment]
you can use docker.

**Comment by ValdikSS** [comment]
In case you're interested in DPI circumvention tool for Windows, try GoodbyeDPI
https://github.com/ValdikSS/GoodbyeDPI
I'm planning to add INTANG circumvention techniques in the closest future.

**Comment by Macronut** [comment]
I wrote a tool for Windows to avoid SNI detection.
https://github.com/Macronut/TCPioneer
